Packs/Sixgill-Darkfeed/Integrations/Sixgill_Darkfeed/Sixgill_Darkfeed_test.py

```diff
@@ -531,3 +531,24 @@ def test_get_indicators_command(mocker):
     submitted_indicators = 0
 
     assert output[2] == expected_ioc_output
+
+
```

Comment:
please consider adding 1 more test case where there is a tag given by the user in the params, and one coming from the feed as an IOC `label`, as the code implies.

Reply:
That is the current test case, as the result already has [ 'compromised', 'shell', 'webshell'] in the "label" and ['tag1','tag2'] are being added to it.

Reply:
I see it in the data, can you add 1 more assert for it in `test_feed_tags`?
So we can check that the tags coming from. the feed are also present. Or am I missing your explanation?